src/main/java/utils/WaitUtils.java

```diff
@@ -2,21 +2,39 @@
 
 import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.FluentWait;
 
 import java.time.Duration;
+import java.util.List;
 
 import static utils.DataPropLoader.getBrowserProperty;
 import static webdriver.DriverManager.getDriver;
 
 public class WaitUtils {
 
+    public static void fWaitWithJSUtil(ExpectedCondition<Object> condition) {
+        getFluentWait().until(condition);
+    }
+
     public static WebElement fWaitUtil(ExpectedCondition<WebElement> condition) {
+        return getFluentWait().until(condition);
+    }
+
+    public static List<WebElement> fWaitElementsUtil(ExpectedCondition<List<WebElement>> condition) {
```

Comment:
Please use generic to make 1 universal method instead of 3 one, which do the same.

P.S Method should be named like fWaitUntil(). Looks like typo